docs/sdk-and-tools/notifier.md

```diff
@@ -276,10 +282,11 @@ Starting with release `v1.2.0`, an additional field `Enabled = true` has been ad
 
 [comment]: # (mx-context-auto)
 
-#### Deduplication
+### Deduplication
 
-If `CheckDuplicates` will be set to true, notifier service will try to connect to a redis 
-instance. In this context, redis will be used as a locker service mechanism for deduplication.
+If `CheckDuplicates` will be set to true in events notifier main config file,
+notifier service will try to connect to a redis instance.
```

Comment:
_the notifier service_

Reply:
Maybe **redis** in bold or italic or as `code`?

Reply:
added